.github/workflows/test-update-git-submodules.yml

```
@@ -107,3 +107,37 @@ jobs:
           test -n "$URL_OUTPUT"
           # url and remoteName must be distinct outputs (regression for original bug)
           test "$URL_OUTPUT" != "$REMOTE_NAME"
+      - name: 🧨 Wipe submodule working tree so init: true must reclone it
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)